cygnus-common/spec/SPECS/cygnus-common.spec

```diff
@@ -171,6 +171,11 @@ rm -rf $RPM_BUILD_ROOT
 /var/run/%{_project_name}
 
 %changelog
+* Wed Sep 17 2019 Fermin Galan <fermin.galanmarquez@telefonica.com> 1.16.0
+- [cygnus-ngsi][doc] Add Install section in README.md (#1576)
```

Comment:
It should be [cygnus-common]. No worth the effort of fixing it for the release branch, but I'll do in a new PR for master branch.

NTC